packages/deployer/package.json

```diff
@@ -7,21 +7,11 @@
   },
   "main": "./dist/bin/deployer.js",
   "scripts": {
-    "_build": "yarn run clean && yarn run compile",
+    "_build": "yarn run clean && yarn run compile && yarn webpack",
     "clean": "rimraf -rf ./dist",
     "compile": "tsc -p tsconfig.build.json",
     "cli": "ts-node bin/deployer.ts",
-    "command:help": "node dist/src/deployer-cli.js --help",
-    "command:deploy": "node dist/src/deployer-cli.js -o $OUTPUT_FILENAME deploy",
-    "command:deploy:nonstop": "node dist/src/deployer-cli.js deploy -o $OUTPUT_FILENAME --nonStop",
-    "command:remove-with-receipt": "node dist/src/deployer-cli.js remove-with-receipt --rf $RECEIPT_FILENAME",
-    "command:remove": "node dist/src/deployer-cli.js remove -a $AIRNODE_ID_SHORT -s $STAGE -c aws -r $REGION",
-    "sls:config": "sls config credentials --provider aws --key $AWS_ACCESS_KEY_ID --secret $AWS_SECRET_KEY",
-    "sls:deploy": "yarn run webpack:copy-config && sls deploy --config serverless.aws.yml --airnodeIdShort $AIRNODE_ID_SHORT --region $REGION --stage $STAGE",
-    "sls:remove": "sls remove --config serverless.aws.yml --airnodeIdShort $AIRNODE_ID_SHORT --region $REGION --stage $STAGE",
-    "sls:invoke": "sls invoke --function startCoordinator --config serverless.aws.yml --airnodeIdShort $AIRNODE_ID_SHORT --region $REGION --stage $STAGE",
-    "webpack": "webpack",
-    "webpack:copy-config": "mkdir -p .webpack/config-data && cp src/config-data/config.json .webpack/config-data/config.json"
```

Comment:
Is this not needed anymore?

Reply:
No, it's not. Serverless was removed, that's the point :smile: All the separate deployer commands can be done with `cli` target and passing the correct arguments. And `webpack:copy-config` is not needed because the configuration file path is configurable in the new recipes so it can be passed from anywhere.